amine/webapp.py

```diff
@@ -1,183 +1,521 @@
-from flask import Flask, render_template, request, jsonify
-from flaskwebgui import FlaskUI
+import sys
+import time
+import sqlite3
+import logging
+import keyboard
+import winsound
+import pyautogui
 import threading
 import webbrowser
-from datetime import datetime, timedelta
-import pyautogui
-import keyboard
-import time
-import sys
+
+import matplotlib

```

Comment:
Please use poetry to add this libraires in your venv, they are not updated in the .toml file